README.md

```diff
@@ -82,11 +82,11 @@ require 'vendor/autoload.php';
 // If you are not using Composer
 // require("path/to/sendgrid-php/sendgrid-php.php");
 
-$from = new SendGrid\Email("Example User", "test@example.com");
+$from = new \Helpers\Email("Example User", "test@example.com");
```

Comment:
Thanks, this is the sort of review I wanted. I didn't care for the Helper namespace, but figured it'd get the conversation going.